tlair/src/main/scala/at/forsyte/apalache/tla/lir/oper/BmcOper.scala

```diff
@@ -1,9 +1,10 @@
 package at.forsyte.apalache.tla.lir.oper
 
 /**
-  * The operators defined in the BMC module. This module give the users a facility to provide hints.
-  * To use this module, the user would have to copy the module somewhere. As our tool is not widely used,
-  * we just hijack some operators during the import stage, e.g., <:
+  * The operators defined in the module Apalache.tla. This module gives the users a facility to provide hints.
+  * The module Apalache is automatically looked up when Apalache is running.
+  *
+  * TODO: rename this class to ApalacheOper, once ik/multicore is merged into unstable.
```

Comment:
Do we tend to open issues to track todos? I'm also happy with a "no" answer here. `grep TODO` is a fine solution :)

Reply:
In this case, it is a convenience thing. Nothing really depends on that name, it is just an inconsistency :)